lib/s3_meta_sync/syncer.rb

```diff
@@ -83,6 +83,18 @@ def download(source, destination)
       end
     end
 
+    def mktmpdir
```

Comment:
this looks sketchy ... better use regular mktmpdir and then handle issues in the ensure if they occur instead of rolling our own